### DISCUSSION TITLE
How to (re)start a job with overrides from `overrides.yaml`?

### DISCUSSION THREAD

**Discussion by greaber** [facebookresearch/hydra#2302] [Q&A] [answered]
Hydra helpfully logs all the overrides used to start a job in an `overrides.yaml` file.  If I want to restart job, I just need to know the config name and the overrides (and the config name is given in `hydra.yaml`.  But restarting a job seems to be a manual process currently: I find the config name in `hydra.yaml` (which is easy) and then I have to paste all the overrides in `overrides.yaml` back into the command-line one by one.  Is there an easier way?

Also, slightly relatedly, what if I wanted to put the overrides in a yaml file to start with (either using the override grammar, like in `overrides.yaml`, or perhaps better as a normal yaml file like hydra uses elsewhere.  If that is possible then I could just stop using so many command-line overrides, and it would be OK too.

**Comment by jieru-hu** [comment, marked as answer]
we've added an experimental feature in Hydra 1.2 https://hydra.cc/docs/experimental/rerun/
note that if you use this callback, you cannot override any value, and the interpolation values will also be exact the same as your previous jobs (for example, timestamps, which you may or may not want.)

you can also use this callback as an example and tweak to make it work for your own use case

**Comment by greaber** [comment]
Thanks, @jieru-hu I'm trying this now
